<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
    <title>PBI Lab</title>
    <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
    <link rel="stylesheet" href="style.css" type="text/css">
    
    
    
    
    <script src="layout/scripts/w3data.js"></script><script type="text/javascript" src="layout/scripts/jquery.min.js"></script><script type="text/javascript" src="layout/scripts/jquery.ui.min.js"></script><script type="text/javascript">
    $(document).ready(function () {
    w3IncludeHTML();
    });
    </script></head>
    
    
    <!-- tabs -->
    
    
    <body id="top">
    <div id="headerContent" class="wrapper col1" style="margin-bottom: -50px;"><meta http-equiv="Content-Type" content="../text/html; charset=iso-8859-1">
    <link rel="stylesheet" href="layout/styles/layout.css" type="text/css">
    <script src="http://www.w3schools.com/lib/w3data.js"></script>
    <script type="text/javascript" src="layout/scripts/jquery.min.js"></script>
    <!-- tabs -->
    <script type="text/javascript" src="layout/scripts/jquery.ui.min.js"></script>
    <script type="text/javascript">
    $(document).ready(function () {
        $("#tabcontainer2").tabs({
            event: "click"
        });
    });
    </script>
    
    <div class="wrapper col1">
      <div id="header" class="clear" style="margin-bottom: 10px; margin-top: -25px;">
        <div class="fl_left">
          <h1 align="center"><a href="index.html"><img src="PBL_logo.png" width="100" height="100" align="left">Protein Bioinformatics Lab</a><a href="http://www.iitm.ac.in"><img src="ism.png" width="100" height="100" align="right"></a>
        <!--img src="images/demo/home-ani.gif" height=50></img--></h1>
        </div>
        <!--div class="fl_right"><a href="#"><img src="images/demo/468x60.gif" alt="" /></a></div-->
      </div>
    </div>
    <!-- ####################################################################################################### -->
    
    </div>
    <!-- ####################################################################################################### -->
    <div class="wrapper col2">
      <div id="container" class="clear"> 
        <table border="1">
          <tbody>
            <tr>
                <th> Name </th>
                <th> Tool Description </th>
                <th> Tool Link</th>
            </tr>
    
        <tr>
            <td><h1>Blast</h1></td>
              <td> 
                <a><p>In bioinformatics, BLAST (basic local alignment search tool) is an algorithm and program for comparing primary biological sequence information, such as the amino-acid sequences of proteins or the nucleotides of DNA and/or RNA sequences.</p></a>
              </td>
              <td>
                <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi"><h1><img src="Blast.png" alt="ANuPP" width="83" height="63"></h1></a>
              </td>
            </tr>
    
        <tr>
            <td><h1>Clustal</h1></td>
              <td> 
                <a><p>Clustal is a multiple sequence alignment program that uses seeded guide trees and HMM profile-profile techniques to generate alignments between three or more sequences.</p></a>
              </td>
              <td>
                <a href="https://www.ebi.ac.uk/Tools/msa/clustalo/"><h1><img src="clustal.jpg" alt="ANuPP" width="83" height="63"></h1></a>
              </td>
            </tr>
    
        <tr>
            <td><h1>EMBOSS Needle</h1></td>
              <td> 
                <a><p>EMBOSS Needle creates an optimal global alignment of two sequences using the Needleman-Wunsch algorithm.</p></a>
              </td>
              <td>
                <a href="https://www.ebi.ac.uk/Tools/psa/emboss_needle/"><h1>EMBOSS Needle</h1></a>
              </td>
            </tr>
    
        <tr>
            <td><h1>EMBOSS Water</h1></td>
              <td> 
                <a><p>EMBOSS Water uses the Smith-Waterman algorithm (modified for speed enhancements) to calculate the local alignment of two sequences.</p></a>
              </td>
              <td>
                <a href="https://www.ebi.ac.uk/Tools/psa/emboss_water/"><h1>EMBOSS Water</h1></a>
              </td>
            </tr>
        <tr>
            <td><h1>Simple Phylogeny</h1></td>
              <td> 
                <a><p>Simple Phylogeny provides access to phylogenetic tree generation methods from the ClustalW2 package.</p></a>
              </td>
              <td>
                <a href="https://www.ebi.ac.uk/Tools/services/web/toolform.ebi?tool=simple_phylogeny"><h1>Simple Phylogeny</h1></a>
              </td>
            </tr>
    
        <tr>
            <td><h1>Genbank</h1></td>
             <td> 
                <a><p>GenBank is the NIH genetic sequence database, an annotated collection of all publicly available DNA sequences</p></a>
             </td>
             <td>
             <a href="https://www.ncbi.nlm.nih.gov/genbank/"><h1>Genbank</h1></a>
             </td>
        </tr>
        <tr>
            <td><h1>ENA</h1></td>
             <td> 
                <a><p>The European Nucleotide Archive (ENA) provides a comprehensive record of the worldâ€™s nucleotide sequencing information, covering raw sequencing data, sequence assembly information and functional annotation.</p></a>
             </td>
             <td>
             <a href="https://www.ebi.ac.uk/ena/browser/search "><h1><img src="ena.jpg" alt="ANuPP" width="83" height="63"></h1></a>
             </td>
        </tr>
        <tr>
            <td><h1>Interpro</h1></td>
             <td> 
                <a><p>InterPro provides functional analysis of proteins by classifying them into families and predicting domains and important sites. </p></a>
             </td>
             <td>
             <a href="https://www.ebi.ac.uk/interpro/"><h1>Interpro</h1></a>
             </td>
        </tr>
        <tr>
            <td><h1>Motif Scan</h1></td>
             <td> 
                <a><p>Motif scanning means finding all known motifs that occur in a sequence. </p></a>
             </td>
             <td>
             <a href="https://myhits.sib.swiss/cgi-bin/motif_scan"><h1>Motif Scan</h1></a>
             </td>
        </tr>
         </tbody></table>
      </div>
    </div>
    </div>
    
    
    </body></html>
